example/pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>com.iopipe</groupId>
       <artifactId>iopipe</artifactId>
-      <version>1.1.0</version>
+      <version>1.2.0-SNAPSHOT</version>
```

Comment:
Sooo the example folder deploys the currently released agent to AWS, which is why it's at this version and not the snapshot

Reply:
Reverted changes.